algorithms/C/linked-lists/merge-two-linkedlists.c

```diff
@@ -0,0 +1,130 @@
+//Merging two Linked-List
+
+#include<stdio.h>
+#include<stdlib.h>
+
+/*Structure definition:
+     data : holds the integer value of the node
+     next : holds the pointer to the next node
+*/
+struct node
+{
+    int data;
+    struct node *next;
+}*la,*lb;                         
+//We have two linked-lists: la (List A) and lb (List B)
+
+
+//Function to create List A of given size
+void lista(int size)
+{
+    int a,i;
+    struct node *temp,*newnode;
+
+    //Head node of List A
+    la=(struct node*)malloc(sizeof(struct node));
+
+    //Getting values to enter into List A
+    printf("\nEnter the value for A: ");
+    scanf("%d",&a);
+    la->data=a;

```

Comment:
What will happen when you will pass 0 as the size of the number of nodes ?

Reply:
I didn't think about that. Even if we pass 0, the program will ask for a value to enter into the list. Thank you for mentioning! I will change my code and solve this problem